install_config/install/advanced_install.adoc

```diff
@@ -2216,22 +2206,35 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 <1> Sets the name for the systemd service.
-<2> Specify the location for your inventory file on your local workstation.
+<2> Specify the location for your local inventory file.
 
 . Use the `systemctl` command to start the installer service as you would any
-other systemd service. This command initiates the cluster installation:
+other systemd service. This command initiates the cluster installation using
+the inventory file specified above and the root user's SSH configuration:
```

Comment:
This is one thing that was not apparent before... ssh config comes from root for this use case (in the docker use case you have to mount it in).